torchao/quantization/quant_api.py

```diff
@@ -460,3 +460,47 @@ def get_per_token_block_size(x):
         return weight
 
     return _get_linear_subclass_inserter(apply_int8_dynamic_activation_int8_weight_quant)
+
+
+def int8_dynamic_activation_int8_semi_sparse_weight():
```

Comment:
does this have similar config as `int8_dynamic_activation_int8_weight`? if so we can add a layout_type arg to that function directly